stac_fastapi/api/stac_fastapi/api/app.py

```diff
@@ -83,10 +84,12 @@ class StacApi:
         converter=update_openapi,
     )
     router: APIRouter = attr.ib(default=attr.Factory(APIRouter))
-    title: str = attr.ib(default="stac-fastapi")
-    api_version: str = attr.ib(default="0.1")
+    title: str = attr.ib(default=os.getenv("STAC_FASTAPI_TITLE", "stac-fastapi"))
+    api_version: str = attr.ib(default=os.getenv("STAC_FASTAPI_VERSION", "0.1"))
     stac_version: str = attr.ib(default=STAC_VERSION)
-    description: str = attr.ib(default="stac-fastapi")
+    description: str = attr.ib(
+        default=os.getenv("STAC_FASTAPI_DESCRIPTION", "stac-fastapi")
+    )
```

Comment:
I'm not quite sure to understand why we set this at the router level and not at the app level?

Reply:
I guess that's something to ask the initial authors?